index.html

```diff
@@ -631,16 +631,16 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
     <p data-lang="en">Books usually use one basic template for page format, whereas magazines often use several templates.</p>
     <p data-lang="zh-hans">书籍多数仅使用一种基础版式体裁，杂志则会使用上数种。</p>
     <p data-lang="zh-hant">書籍多數僅使用一種基礎排版體裁，雜誌則會使用上數種。</p>
-    <p data-lang="en">Although in books, there tends to be one template for the page format, some further design effort based around the basic page format will be needed for pages such as the table of contents and indexes. Furthermore, there are many examples of indexes with a different page format than the basic page format, and vertically set books often have indexes in horizontal writing mode, and sometimes multiple columns. However, while doing actual design for the page, the size of the type area should not exceed the basic page template.<span class="issue"> (some doubts about the translation of 版面 here)</span>.</p>
+    <p data-lang="en">Although books tend to use one template for its page format, some further design effort based will be needed to extend that template for pages such as the table of contents and indexes. Furthermore, there are many examples of indexes with a different page format than the basic page format, and vertically set books often have indexes in horizontal writing mode, and sometimes multiple columns. However, while doing actual design for the page, the size of the type area should not exceed the basic page template.<span class="issue"> (some doubts about the translation of 版面 here)</span>.</p>
```

Comment:
Hmm, this one went a bit haywire.  
'for its page format' -> 'for their page format'
'some further design effort based will be' -> 'some further design effort will be'